src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriterILGen.cs

```diff
@@ -1380,6 +1384,53 @@ private void WriteAttribute(SourceInfo source, AttributeAccessor attribute, stri
             }
         }
 
+        void WriteNullableMappingForWriteAttribute(string name, string ns, SourceInfo source, TypeMapping mapping)
+        {
```

Comment:
I usually try to stay out of naming discussions as they can get a bit too religious but this name is a little too long and feels wrong. Maybe `WriteNullableAttribute`